packages/html/src/upload/upload-file.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { Button } from '../button';
+import { Icon } from '../icon';
+import { classNames } from '../utils';
+import { UploadFileInfo } from './upload-file-info';
+
+export interface UploadFileProps {
+    className?: string;
+    fileStatus?: 'error' | 'progress' | 'success';
+    progress?: number;
+    fileName?: string;
+    fileIcon?: string;
+    fileSize?: string;
+    fileState?: 'uploaded' | 'uploading';
```

Comment:
do we need the file prefix here ? As we are already in the UploadFile context, we can use name, size, etc.